examples/scale/scenarios/firstmatch/firstmatch.go

```diff
@@ -50,7 +51,11 @@ func (_ *FirstMatchScenario) Ticket() *pb.Ticket {
 	return &pb.Ticket{}
 }
 
-func (_ *FirstMatchScenario) MatchFunction(p *pb.MatchProfile, poolTickets map[string][]*pb.Ticket) ([]*pb.Match, error) {
+func (_ *FirstMatchScenario) Backfill() *pb.Backfill {
```

Comment:
in this case, if the "instance" for FirstMatchScenario is not used, simply set (*FirstMatchScenario) as owner of the func. I think is cleaner